app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from "react";
+import { fetchGreeting } from "./redux/greetingStore";
+import { useDispatch, useSelector } from "react-redux";
+
+export default function Greeting() {
+    const dispatch = useDispatch();
+    const message = useSelector((state) => state.message);
+    useEffect(() => {
+        dispatch(fetchGreeting());
+    }, []);
+    return (
+        <div>
+            <h1>Greeting: {message.greeting}</h1>
+        </div>
+    );
+}
```

Comment:
- [ ] Kindly ensure your app actually works by displaying a random message when visiting the root path based on the requirements.

Reply:
This happens from the api get response each time you call it gives a random message